### DISCUSSION TITLE
Add ability to compare by content inside ZIP files

### DISCUSSION THREAD

**Discussion by lars18th** [doublecmd/doublecmd#1584] [Suggestions]
Hi,

For years I'm using TC and one of the functionalities that I'm using daily is: "Synchronize directories and Compare by content" when one of the panel is a ZIP file. In TC this functionality is implemented in a very simple way: unpack the ZIP file in the TEMP directory and then compare the contents with the other panel.

But this is missing in DC... when synchronising dirs where one is a ZIP file, in this case the "by content" checkbox is greyed out and unselectable.

I understand that this could be slow with large ZIP files. Therefore I suggest to add a new configuration flag to enable/disable this support in a simple way.

I feel this request is similar to #1111. And I hope that @alexx2000 will want to implement it.
Thank you.

**Comment by ciros88** [comment]
+1 total commander is able to do dirs synchronization with archives in both panels, and yeah, it misses a lot for my daily use

**Comment by ags1234** [comment]
+1

**Comment by lars18th** [comment]
Why this is not already implemented?
This is implemented when **ONE panel** is a ZIP file, but **not when TWO panels are ZIPs**.

**Comment by lars18th** [comment]
Hi,

We are still waiting for the synchronization of panels within ZIP files with CONTENT comparison.
Please!!! 🙏
